build.gradle

```diff
@@ -51,6 +53,15 @@ subprojects {
         }
 
         alpnboot_package_name = 'org.mortbay.jetty.alpn:alpn-boot:' + alpnboot_version
+
+        def exeSuffix = Os.isFamily(Os.FAMILY_WINDOWS) ? ".exe" : ""
+	// The split is to workaround everything after the colon in C:\ being
```

Comment:
No, there is not a tracking bug. I'm pretty certain the problem is simply using 1-arg split() instead of 2-arg split(). I was going to send a PR, but had trouble getting my modified version to actually be used.